api/src/main/java/io/opencensus/internal/NoopScope.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2017, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.internal;
+
+import io.opencensus.common.Scope;
+
+/**
+ * A {@link Scope} that does nothing when it is closed.
+ */
+public final class NoopScope implements Scope {
+  private static final Scope INSTANCE = new NoopScope();
```

Comment:
Let's decide on this and keep it consistent. Should we make INSTANCE public or always have function calls? Advantage when directly access variable is clear that is not creating a new instance, disadvantage is that is harder to implement any side effect (or deprecate).